pyi.py

```diff
@@ -430,12 +432,16 @@ def visit_AsyncFunctionDef(self, node: ast.AsyncFunctionDef) -> None:
     def _Y019_error(
         self, node: ast.FunctionDef | ast.AsyncFunctionDef, typevar_name: str
     ) -> None:
-        cleaned_method = deepcopy(node)
-        cleaned_method.decorator_list.clear()
-        new_syntax = re.sub(
-            fr"(\W){typevar_name}(\W)", r"\1Self\2", ast.unparse(cleaned_method)
-        ).replace("\n   ", "")
-        error_message = Y019.format(typevar_name=typevar_name, new_syntax=new_syntax)
+        error_message = Y019.format(typevar_name=typevar_name)
+
+        if sys.version_info >= (3, 9):
+            cleaned_method = deepcopy(node)
+            cleaned_method.decorator_list.clear()
+            new_syntax = re.sub(
+                fr"(\W){typevar_name}(\W)", r"\1Self\2", ast.unparse(cleaned_method)
+            ).replace("\n   ", "")
+            error_message += f", e.g. `{new_syntax}`"
```

Comment:
This plugin's convention seems to be double quotes instead of backticks:

```suggestion
            error_message += f', e.g. {new_syntax}"'
```